docs/src/main/asciidoc/logging.adoc

```diff
@@ -545,6 +544,9 @@ quarkus.log.socket.enable=true
 quarkus.log.socket.endpoint=localhost:4560
 ----
 
+Typically, use this handler with the `quarkus-logging-json` extension to send logs in ECS format to an Elasticsearch instance.
```

Comment:
"Typically, [you] use this handler" sounds alien to [non-native] me. How about "Typically, this handler is used ..."?

Reply:
OK. Will do.